### DISCUSSION TITLE
composion of the hourly wage

### DISCUSSION THREAD

**Discussion by katWei1** [kimai/kimai#3822] [Questions & Answers]
How is the hourly wage composed? It does not match the hourly wage entered for the project (which is always set to 0), nor does it match that of the employees (each has an individual entry here). Nevertheless, in the project summary there is an incomprehensible number at this point. Please help to understand, the payroll department gets completely confused with it.

**Comment by kevinpapst** [comment]
Read the documentation about rates: https://www.kimai.org/documentation/rates.html
